readme.md

```diff
@@ -48,11 +48,13 @@ data/
     manatoges.yaml
 ```
 
-Inside my view templates:
+Inside my view templates I can use the following line to inspect the `data` object provided by roots-yaml:
 
-`data.staff` returns the data in `data/staff.yaml`
+```jade
+pre=JSON.stringify(data, null, 2)
```

Comment:
It's a readability thing.
This gives you the object printed out with indentation and line breaks whereas a single param gives you a one-liner of the entire object.